Maskbook/Scene/Wallet/TokenDetail/View/TransactionCell.swift

```diff
@@ -337,8 +351,22 @@ extension TransactionCell {
             guard let gasLimit = txToSpeedUp.transactionInfo?.gaslimit else { return }
             
             let token  = WalletAssetManager.shared.getDefaultMainToken()
+            
+            var gasNetModel = txToSpeedUp.transactionInfo?.gasNetModel
+
+            if let maxFeePerGas = txToSpeedUp.transactionInfo?.gasNetModel?.suggestedMaxFeePerGas{
+                let gwei = NSDecimalNumber(string: maxFeePerGas).multiplying(by: NSDecimalNumber(string: "1.5"))
```

Comment:
Init a NSDecimalNumber with a `number` would be better than using a `string` IMO